pgmpy/tests/test_models/test_MarkovModel.py

```diff
@@ -216,12 +216,12 @@ def test_factor_graph(self):
 
         factor_graph = self.graph.to_factor_graph()
         self.assertIsInstance(factor_graph, FactorGraph)
-        self.assertListEqual(sorted(factor_graph.nodes()),
-                             ['Alice', 'Bob', 'Charles', 'phi_Alice_Bob',
-                              'phi_Bob_Charles'])
-        self.assertListEqual(hf.recursive_sorted(factor_graph.edges()),
-                             [['Alice', 'phi_Alice_Bob'], ['Bob', 'phi_Alice_Bob'],
-                              ['Bob', 'phi_Bob_Charles'], ['Charles', 'phi_Bob_Charles']])
+        self.assertListEqual(factor_graph.nodes(),
```

Comment:
@VSSILPA  You can convert both lists to python set. Order of elements might change in list so test might fail.

Reply:
Will recursive_sorted() function work in case of elements with different data types? Is it fine to just write factor_graph.edges() instead of recursive_sorted(factor_graph.edges())?